models/entry.js

```diff
@@ -1,10 +1,11 @@
-import { Schema } from 'mongoose';
+import mongoose, { Schema } from 'mongoose';
 import constants  from '../src/constants';
 
-const Entry = new Schema({
+const entrySchema = new Schema({
+  id: String,
   item_id: { type: Schema.ObjectId, ref: 'Item' },
   status: { type: String, default: constants.OPEN },
   comment: String
 });
 
-export default Entry;
+export default mongoose.model('Entry', entrySchema);
```

Comment:
Probably because I wasn't calling any of the mongoose methods on entry, i.e. in the seed file. This is better, though.